### PR TITLE
CB: Emit unimplemented terms as PlannerError

### PR DESCRIPTION
**Pull request by drostron**

### CODE REVIEW

File: couchbase/src/main/scala/quasar/physical/couchbase/planner/QScriptCorePlanner.scala [hunk 1]
```diff
@@ -215,7 +219,7 @@ final class QScriptCorePlanner[F[_]: Monad: NameGenerator, T[_[_]]: Recursive: C
     case qscript.Subset(src, from, op, count) => op match {
       case Drop   => takeOrDrop(src, from, count.right)
       case Take   => takeOrDrop(src, from, count.left)
-      case Sample => ???
+      case Sample => unimplementedP("Sample")
```
Comment (wemrysi) [inline couchbase/src/main/scala/quasar/physical/couchbase/planner/QScriptCorePlanner.scala]: I think all the other planners just implement this as `Take` currently, if you want to hop on that bandwagon.

**Comment by wemrysi** [comment]
LGTM, let me know if you want to update the plan for `Sample`, otherwise I'll merge.

File: couchbase/src/main/scala/quasar/physical/couchbase/planner/QScriptCorePlanner.scala (hunk 1 above)
Comment (wemrysi) [inline couchbase/src/main/scala/quasar/physical/couchbase/planner/QScriptCorePlanner.scala]: Up to you, not sure if it is actually possible for this to show up in a plan yet...

Comment (sellout) [inline couchbase/src/main/scala/quasar/physical/couchbase/planner/QScriptCorePlanner.scala]: Yeah, can't actually sample via SQL2 or even LP yet.

Comment (drostron) [inline couchbase/src/main/scala/quasar/physical/couchbase/planner/QScriptCorePlanner.scala]: Ok, I'll leave it as is for now then.

**Comment by drostron** [comment]
@wemrysi decided to leave `Sample` as is so should be ready for merge.

**Pull request merged**